ui/src/variables/selectors/index.tsx

```diff
@@ -48,3 +54,29 @@ export const getValueSelections = (
 
   return selections
 }
+
+const getVariableAssignmentsMemoized = memoizeOne(
+  (
+    valuesState: ValuesState,
+    variablesState: VariablesState
+  ): VariableAssignment[] => {
+    if (!valuesState) {
+      return []
+    }
+
+    return Object.entries(valuesState.values).map(([variableID, values]) => {
+      const variableName = variablesState[variableID].variable.name
```

Comment:
what do you think about using a `get` here?

Reply:
👍